variables.tf

```diff
@@ -367,3 +367,9 @@ variable "fargate_pod_execution_role_name" {
   type        = string
   default     = null
 }
+
+variable "service_ipv4_cidr" {
```

Comment:
Can you please rename this into `cluster_service_ipv4_cidr` for namming consistency.